decidim/decidim
# Fix filter URL not updated with the text search input

#### :tophat: What? Why?
When the filter forms are submitted manually by pressing the "search" button or enter within the search input, i.e. when doing a text search, the URL is not updated. This fixes the issue.

#### :pushpin: Related Issues
- Fixes #7433

#### Testing
- Go to a component with search (e.g. debates, proposals, etc.)
- Type in something in the text search input
- Press enter
- Check if the current URL is updated
